bin/offline-cluster.sh

```diff
@@ -47,9 +47,7 @@ ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/seed-offline-containerd.yml
 ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/sync_time.yml -v
 
 # Run the rest of kubespray. This should bootstrap a kubernetes cluster successfully:
-ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/kubernetes.yml --skip-tags bootstrap-os,preinstall,container-engine
-
-./bin/fix_default_router.sh
```

Comment:
I cannot find where this script is called now. Am I'm blind (probably) or is `fix_default_router.sh` now unused?

Reply:
this script is no longer needs to be executed, there was some issue with dns configs on k8s v1.23.7 which is no longer found in this newer version.